src/com/dakl/Main.java

```diff
@@ -91,13 +95,15 @@ static void threeByThree(Team center_team, Team[] surrounding, Space center, boo
     public static boolean inputIsValid(CheckerBoard game, String input) {
         String[] command = input.split(" ");
 
+        // REVIEW : Variable java naming convention
         Space unvalidated_pc_loc = null;
         Space unvalidated_end_space = null;
```

Comment:
By default, java's variable naming convention is camelCase for member variables. When you're in Rome you do like the romans :)